source/sass/views/initiatives.scss

```diff
@@ -5,3 +5,15 @@
 .bottom-line {
   border-bottom: 1px solid $light-gray;
 }
+
+// Fellowships--Support
+
+.fellow-support {
+
+  &__intro {
```

Comment:
we're not using any underscores in css, I'd recommend staying with the "kebab case" of a single hyphen

Reply:
Sorry, I'm using BEM naming conventions. I'll update them, but I'll also open up a ticket to discuss sass naming conventions further.

Reply:
See #2544